catalog/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path
+from catalog.apps import CatalogConfig
+from catalog.views import home
+from catalog.views import contacts
+
+
+app_name = CatalogConfig.name
+
+urlpatterns = [
+    path("", home, name="home"),
+    path("", contacts, name="contacts")
```

Comment:
У тебя не могут два пути вести на разные вьюшки
```suggestion
    path("contacts/", contacts, name="contacts")
```